### PR TITLE
Added wggs for my school's club

### PR DESCRIPTION
**Pull request by Luna5379**
A club website for my school's club!

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -3051,3 +3051,7 @@ gps:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+wggs:
+  - ttl: 600
+    type: CNAME
+    value: https://luna5379.github.io.
```
Comment (alx-alexpark) [inline hackclub.com.yaml:3057]: Hi, could you sort this alphabetically within the file?

**Review by alx-alexpark — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 1 above)
Comment (alx-alexpark) [inline hackclub.com.yaml:3057]: CNAME records can't have protocol or path
```suggestion
    value: luna5379.github.io
```

Comment (Luna5379) [reply hackclub.com.yaml:3057]: yes ill do that now

Comment (Luna5379) [reply hackclub.com.yaml]: done!

**Review by alx-alexpark — APPROVED** [review]
h

**Pull request merged**